pages/cart.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>E-commerce-app | My Cart</title>
+    <link rel="stylesheet" href="https://zeta-ui.netlify.app/components/root.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" />
+    <link rel="stylesheet" href="../app.css">
+
+</head>
+
+<body>
+    <!-- header starts  -->
+    <header class="header-wrapper">
+        <a class="active" href="../index.html">
+            <h1>BookzKart</h1>
+        </a>
+        <form action="" class="search-container">
```

Comment:
Action attribute is empty, you can remove it, because we are not submitting the form